runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/base/src/main/hilla-scaffold-package-name-holder.txt

```diff
@@ -0,0 +1,5 @@
+// This file is used only to execute HillaAppInitUtility.scaffold and generate
+// the source code to be included in the codestart
+// @SpringBootApplication
```

Comment:
Why is SpringBootApplication commented out?

Reply:
Because Hilla scaffolding utility looks for a file with that annotation to get the base package of the application

Reply:
Ok, can you add this to the technical overview, since this could be more compatible?

Reply:
Is some of this file generated or where does it has to be present or for what task?

Reply:
This file is in only on the codebase and it is required only for generating/updating the other files in the codestart.
It is ignore in JAR creation